apiserver/JobSubmission.md

```diff
@@ -0,0 +1,211 @@
+# Job Submission using API server
+
+Ray provides very convinient and powerful [Job Submission APIs](https://docs.ray.io/en/latest/cluster/running-applications/job-submission/rest.html). The issue is that it needs to access cluster's dashboard Ingress, which currently does not have any security implementations. Alternatively you can use Ray cluster head service for this, but in this case your Ray job management code has to run in the same kubernetes cluster as Ray. Job submission implementation in the API server leverages already exposed URL of the API server to locate cluster and use its head service to implement Job management functionality. Because you can access API server running on the remote kubernetes cluster you can use these APIs for managing remote Ray clusters without exposing them via Ingress.
+
+## Using Job Submission APIs
+
+Note that job submission APIs will only work if you are running API server within kubernetes cluster. Local Development option of the API server will not work.
+
+The first step is to deploy KubeRay operator and API server.
+
+### Deploy KubeRay operator and API server
+
+Reffer to [readme](README.md) for setting up KubRay operator and API server.
+
+```shell
+make docker-image cluster load-image deploy
+```
+
```

Comment:
This instruction is good for development such as testing the PR.
```suggestion
For development, we can build the API server from source code and deploy to the local KinD cluster.
```shell
make docker-image cluster load-image deploy
```

```

Reply:
fixed